DuckDuckGo/MainViewController.swift

```diff
@@ -1433,6 +1434,11 @@ class MainViewController: UIViewController {
                 tunnelDefaults.enableEntitlementMessaging()
             }
 
+            if await controller.isConnected {
+                DailyPixel.fireDailyAndCount(pixel: .privacyProVPNBetaStoppedWhenPrivacyProEnabled,
```

Comment:
Since we know that the only source of these pixels will be the browser, we can likely remove the source parameter overall. What do you think?

Reply:
Agreed 👍